### ISSUE TITLE
Status of project?

### ISSUE THREAD

**Issue by CarsonRedeye** [michallaskowski/kuiks#9]
Hi I'm interested in this project. I would like to know if you are using this at your work, or if this is just experimental?
I like the idea of using the same tools to test that we are familiar with, rather than learn a new tool like Appium. Also it seems like it closer to using the native tools and would be easy to customise if we needed particular features of either XCTest or Espresso.
However I'm hesitant to start using it, as I would be asking QA team to help write the tests and they don't have android experience. What would you recommend?

**Comment by michallaskowski** [comment]
Hi. I am not using it currently, hopefully at a point where I have more time for my apps that I am writing. 
I would treat the project as under development, awaiting some eager users that would feel comfortable to contribut to it, maybe co-own it if they want.
I checked some amount of test cases when developing it, but of course more changes can pop-up. And XCTests and Espresso are still evolving, so there may be some additions needed.
Long story short - if you are looking for something stable and mature, not here yet.
If you want to risk it and invest in it, to still save some time for your QA team, and potentially help improving the project - that would be great.

**Comment by CarsonRedeye** [comment]
Thanks for the info. I would love to try to use it, but I think it could be difficult to sell it to our QA team. Maybe I'll try to use it as the developer's test platform, in conjunction with QA's Appium tests.

**Comment by CarsonRedeye** [comment]
I notice that the current code doesn't work on xcode 12. Would you like me to submit a pr to use XCFrameworks and upgrade gradle version?

**Comment by michallaskowski** [comment]
@CarsonRedeye if you have time for it, feel free :) I may be busy till the end of the week, but would try to find time to review and test it.
